mantidproject/mantid
# Store information about detector trajectories in MDNorm

**Description of work.**
This algorithm will store information about detector trajectories in reciprocal space. 
**Report to:** [user name]/[nobody]. 

**To test:**
Code review and see usage example.

Part of #23320 .

*This does not require release notes* because they will be provided in a later algorithm (see associated issue)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
